config.toml

```diff
@@ -25,9 +25,43 @@ color = "#000000"
 # Optional. Only used when updating from GitHub releases
 genesis_hash = "0xb0a8d493285c2df73290dfb7e61f870f17b41801197a149ca93654499ea3dafe"
 
+[[chains]]
```

Comment:
what about add colors from polkadot js? Gray color looks like unsupported

https://github.com/polkadot-js/apps/blob/b80734b80914101be38a1484456678f2b4765ae1/packages/apps-config/src/ui/colors.ts

Reply:
@Asmadek sorry, I don't understand where is the problem.